schemas/json/layout/layout.schema.v1.json

```diff
@@ -900,6 +906,16 @@
           "type": "string",
           "title": "Binding to show in summary",
           "description": "The value of this binding will be shown in the summary component for the list. This binding must be one of the specified bindings under dataModelBindings."
+        },
+        "mapping": {
+          "type":"object",
+          "description": "Usally used for search",
+          "examples": [
+            {
+              "some.source.field": "key1",
+              "some.other.source": "key2"
+            }],
+          "title": "Mapping"
```

Comment:
```suggestion
          "$ref": "#/definitions/mapping",
          "description": "(Optional) Used to map data model paths to query parameters when fetching list data"
```

Reply:
We already have a definition for mapping used elsewhere, so it's preferable to re-use that.